terraform_alert_config/import_adapter/healthcare.tf

```diff
@@ -0,0 +1,16 @@
+# Healthcare processing
+# create healthcare dataset for storing dicom files
+resource "google_healthcare_dataset" "dataset" {
+  name     = var.dataset
+  location = var.region
+}
+
+# create healthcare store for storing dicom files
+#resource "google_healthcare_dicom_store" "default" {
+#  name    = var.store
+#  dataset = google_healthcare_dataset.dataset.id
+
+#  labels = {
+#    label1 = "healthcare-label"
+#  }
+#}
```

Comment:
Will terraform recreate these stores/datasets if they already exist? If not I think we should have one for the dataset and one for the dicom store

Reply:
now a new dataset and store are being created, if they exist, then during creation you will receive an error that the dataset / store already exists. Do we need to create another audited dataset and store?

Reply:
No, I think just one store is fine. I just thought it would make more sense to create both a dataset and store in the terraform config so the user wouldn't have to create them separately.